blog/_posts/2017-07-04-pivotal-london-intro.md

```diff
@@ -0,0 +1,57 @@
+---
+title: And so it begins!
+class: page blog
+layout: bootstrap
+current: blog
+permalink: blog/2017-07-04-and-so-it-begins
+author: Emma Deacon and Kara de la Marck
+twitter: EmmaMDeacon KaraMarck
+categories:
+- student-posts
+
+---
+
+
+
+Hi, we’re [Emma](https://twitter.com/EmmaMDeacon) and [Kara](https://twitter.com/KaraMarck), the members of Pivotal, London! We’re working on [Babel](https://github.com/babel/babel) the JavaScript language compiler, with the help of our incredible mentor [Henry Zhu](https://twitter.com/left_pad), our supportive supervisor [Ines](https://twitter.com/ines_opcoelho), and our amazing hosts [Pivotal](https://twitter.com/pivotal). We’re so lucky to be in this position, and we cannot thank Rails Girls Summer of Code enough!!
+
+![Alt Text](https://media.giphy.com/media/k3kqJ2d8cUvSM/giphy.gif)
+
+***so fabulous***
+
+![Alt Text](https://media.giphy.com/media/y4PQTcLTYJwOI/giphy.gif)
+
+We met through [Founders and Coders](https://twitter.com/founderscoders), and we were joined together by fate ❤️ Hahah no, not really, we were paired together during one of the paired programming sessions on Object Oriented Programming, and since then, we’ve always be involved in coding and being involved in the women in technology scene. We are deeply and passionately involved in [codebar](https://twitter.com/codebar?lang=en-gb) and a huge number of other coding meetups and coding classes in and around London - if you ever want to know about which ones to attend if you’re in London, we’ve got tonnes to recommend! :)
+
+Emma is currently doing her MSc in Technology Entrepreneurship at UCL, and Kara was working as a developer at Ticketmaster before joining Rails Girls Summer of Code. Both of us have been involved in smaller team projects through Founders and Coders, but this is the first time we have been heavily involved in Open Source projects.
+
+One of the best things about working somewhere like Pivotal is the incredibly open and friendly environment (as well as the amazingggg breakfasts 😍 😍), and despite having only been here two days, we have already learnt a tonne about Babel and compilers in general. We’re really lucky to have so many coaches, who are truly interested and passionate about promoting women in technology, and women in open source. We’ve had two informative and engaging lunch and learns with them (and we’ve haven't even been here two full days!), where we’ve explored working in a technology company, and the scope of future technology teams.  
+
+And now... we’re pleased to introduce you to our coaches!!
+
+![Alt Text](https://media.giphy.com/media/MJVEi3fmDjd1S/giphy.gif)
```

Comment:
Hi Emma, sorry to bug you again but it seems like the giphy images/gif are still shown with a direct link to giphy rather than added to the `img` folder. could you still do that? You'd just need to save each one of them to the folder and push that change. we can merge (and share the post!!!) after that <3